Obleak.Fody/Extensions.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Mono.Cecil.Rocks;
+
+namespace Obleak.Fody
+{
+    internal static class Extensions
```

Comment:
I agree up to a point - it's annoying enough not having this as part of the .Net framework (I understand their justification), the idea of having this sole extension method in another file would make me :rage4: 

If there were to ever be another extension method added to Extensions I'd move it out to a separate .cs (originally I had this in the .Core project in it's own file)